commit a1a52032d5bbf5011b47993d1a509552cc9baed6 (refs/changes/01/5101/2)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2018-11-22T17:42:59+00:00 (11 months ago)
    
    MANTA-4022 propeller's nodeunit dep cannot install with old node (fix build)
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index 2e659ef..fe9a794 100644
--- a/Makefile
+++ b/Makefile
@@ -32,7 +32,7 @@ NPM             := npm
 # Files
 #
 DOC_FILES        = $(shell find docs -name '*.md' | cut -d '/' -f 2)
-JS_FILES        := $(shell ls *.js) $(shell find lib test bin -name '*.js')
+JS_FILES        := $(shell ls *.js) $(shell find lib bin -name '*.js')
 JSL_CONF_NODE    = tools/jsl.node.conf
 JSL_FILES_NODE   = $(JS_FILES)
 JSSTYLE_FILES    = $(JS_FILES)
